images/module-dependencies.dot

```diff
@@ -0,0 +1,8 @@
+digraph {
+  Update -> Main
+  View -> Main
+  Model -> Main
+  Model -> Update
+  Model -> View
+  Flags -> Main
+}
```

Comment:
💭  I thought the png came out nicer when Main was at the top instead of at the bottom... Is there any way to force dot to do that?